src/Symfony/Component/ExpressionLanguage/Node/ConstantNode.php

```diff
@@ -20,11 +20,14 @@
  */
 class ConstantNode extends Node
 {
+
     private bool $isIdentifier;
+    public bool $isNullSafe;
```

Comment:
public should be first, and blank line above should be remove
let's declare it as readonly also

Reply:
`public readonly bool $isNullSafe;`